mining/mining.go

```diff
@@ -739,7 +740,7 @@ mempoolLoop:
 		// save the fees and signature operation counts to the block
 		// template.
 		blockTxns = append(blockTxns, tx)
-		blockSize += txSize
+		blockSize = blockPlusTxSize
```

Comment:
This is only tangentially related. Can revert if you prefer. It just has potential for doing decision making with one number and then storing a different number.

Reply:
I think this looks correct